### DISCUSSION TITLE
Is there any plan to release the next version after 2.3.1?

### DISCUSSION THREAD

**Discussion by tpd2022** [quickfix-j/quickfixj#655] [Q&A] [answered]
I noticed that there hasn't been a new release since version 2.3.1 was released in November 2021. I would like to know if there are any plans to release version 2.3.x or version 3.0, and if so, when can we expect it? Thans.

**Comment by chrjohn** [comment]
There are plans for 3.0 but no fixed date. Is there something special that you need? When it is no big change then maybe it makes sense to build your own version.

**Reply by tpd2022** [reply]
Thanks for your reply :)
I am investigating Java implementations for the FIX protocol. I believe Quickfix/J is suitable for most scenarios, so I would like to know its release plan in advance.

**Reply by chrjohn** [reply, marked as answer]
There is no fixed plan but I think for most users QFJ should contain everything that is needed. Of course there may be special requirements that are not covered. 
There is one big change that still needs to go into 3.0.0 (#460) but after that I expect that pull requests will be merged more frequently again. IMHO there is no change in 3.0.0 that should keep you from using 2.3.1.
If you still have questions, feel free to ask.

**Reply by tpd2022** [reply]
Thanks for your help :)
